osu.Game/Overlays/Profile/Header/Components/GroupBadge.cs

```diff
@@ -68,7 +71,15 @@ private void load(OverlayColourProvider? colourProvider, RulesetStore rulesets)
                         }
                     }
                 }
-            });
+            };
+
+            AddRangeInternal(groupDrawables);
```

Comment:
There's no need for putting drawables into a variable now, revert back to `AddRangeInternal(new Drawable[] ...);`